.gitlab-ci.yml

```diff
@@ -158,11 +158,13 @@ prepare:back:
   dependencies: []
   artifacts:
     name: "$CI_COMMIT_REF_NAME:$CI_COMMIT_SHA:back"
-    expire_in: 1d
+    expire_in: 12 hrs
```

Comment:
let's configure it with variable to edit in single place, OTOH there should be a setting in UI

Reply:
tried, gitlab ci doesn't support variable for artifacts TTL